### ISSUE TITLE
CFF2 table update

### ISSUE THREAD

**Issue by josh-hadley** [MPEGGroup/OpenFontFormat#48]
[Laurence Penney](https://github.com/Lorp), in cooperation with Adobe and with guidance from Vlad Levantovsky and Peter Constable, has prepared an overhauled CFF2 table specification described as:
> This is proposed as an updated specification for the CFF2 table in OpenType fonts. The updated proposal fixes errors, has more explanations, clearer tabulation, and has involved restructuring and rewriting the entire document. It also removes dependencies on other documents, notably the Adobe Type 1 specification.
>
> We hope that its publication will make CFF2 fonts significantly easier to understand for developers of font tools, font rasterizers, and any other code that deals with binary fonts.

This is a work in progress; the current **DRAFT** is available to review ahead of the formal submission process here: https://github.com/MPEGGroup/OpenFontFormat/pull/49

**Comment by simoncozens** [comment]
I have some feedback to this. Can I suggest it be submitted as a pull request (not intending to be merged) as a way for people to add line-by-line feedback?

**Comment by josh-hadley** [comment]
I defer to @vlevantovsky on that question; I was advised specifically to create an issue for this. I do appreciate that a PR would make it easier to do quoted reviews.

**Comment by vlevantovsky** [comment]
Since we are discussing this for the purposes of fine-tuning an input contribution, I think it would be fine to do it as a PR. We cannot host output documents online (unless they're approved as public and then placed in a specific location) but as far as input contributions are concerned, until they are submitted and accepted the copyrights belong to authors.

**Comment by josh-hadley** [comment]
no-merge PR with updated content: https://github.com/MPEGGroup/OpenFontFormat/pull/49

**Comment by vlevantovsky** [comment]
Thank you @josh-hadley. 
All, please note that the document uploaded for review is a draft of the input contribution to be submitted by a member of this AHG for consideration at the next WG meeting - something that we'll have a chance to discuss at a later time as part of the formal review process, if the proposed changes are adopted. 
Your comments and suggested improvements would be very much appreciated; however, the member preparing this contribution (Adobe) is under no obligation to adopt them as part of this input submission. Please also note that if your comments are incorporated as part of the input contribution, the copyrights for that contribution will still belong to Adobe.

**Comment by josh-hadley** [comment]
Pinging @simoncozens — as you requested, a PR has been created (#49) to enable line-by-line feedback, if you're still interested. The proposal will be submitted for the formal review process soon.

**Comment by simoncozens** [comment]
Thank you - I am interested, and I know time is short, but I have been busy with other things. I only really have editorial rather than technical changes.
